### PR TITLE
[DDW-1026] Add Delegation Status of Wallets in Delegation Setup Wizard

### PR DESCRIPTION
**Pull request by MarcusHurney**
This PR adds the delegation status (Stake Pool ticker) for each wallet in the `Select` component's dropdown options when selecting a wallet from which to delegate in the delegation setup wizard.

## Screenshots
Screen Shot 2019-12-02 at 12 50 52
Screen Shot 2019-12-02 at 12 51 02
Screen Shot 2019-12-02 at 12 52 06
Screen Shot 2019-12-02 at 12 52 16
Screen Shot 2019-12-02 at 12 53 32

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1575302264218700)
- [ ] WALLETS > Transfer Funds > [Step 1](https://deploy-preview-1725--daedalus-storybook.netlify.com/?path=/story/wallets-transfer-funds--step1#themeName=Cardano&localeName=English&osName=Windows)
- [ ] DECENTRALIZATION > Wizard > [Delegation Wizard](https://deploy-preview-1725--daedalus-storybook.netlify.com/?path=/story/decentralization-wizard--delegation-wizard#themeName=Cardano&localeName=Japanese&osName=Windows)

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by MarcusHurney** [comment]
@nikolaglumac 
I don't see anywhere in the staking or wallets stores where it can be determined which wallet is currently delegating to which pool. Because of this, I built out the UI as far as possible, but it cannot be finished unless there is more data available indicating the relationship between the user's wallets and the pools being delegated to. On the UI, I implemented the correct color calculation based on a pool's rank and use one of the pools we have in the staking store to render the slug next to the wallet name and balance in the dropdown.

Screen Shot 2019-11-26 at 6 50 51 PM

**Comment by nikolaglumac** [comment]
Thanks @MarcusHurney - we will take it from here.

**Comment by nikolaglumac** [comment]
@daniloprates can you check the flow issues?

**Comment by daniloprates** [comment]
@nikolaglumac @MarcusHurney I've changed a bit the logic, so the dropdown only receives the wallets list. The `getWallets` API is returning wallets with fake Stake Pools, for now, so it's easy to track what needs to be removed once the Stake Pools API is done.

File: storybook/stories/_support/StoryProvider.js [hunk 1]
```diff
@@ -88,6 +93,38 @@ export const WALLETS = [
   },
 ];
 
+export const WALLETS_V2 = [
```
Comment (tomothespian) [inline storybook/stories/_support/StoryProvider.js]: @daniloprates maybe would be better to name this like `SHELLEY_WALLETS`. What do you think about that?

Comment (nikolaglumac) [reply storybook/stories/_support/StoryProvider.js]: I would keep it WALLETS_V2 for now. We don't use shelley anywhere...
cc @daniloprates @tomothespian

Comment (tomothespian) [reply storybook/stories/_support/StoryProvider.js]: @nikolaglumac Deal, approving

**Comment by a-rukin** [comment]
image
please reduce this padding from 10px to 6px

**Comment by daniloprates** [comment]
> please reduce this margin-right from 10px to 6px

@a-rukin done!

**Comment by a-rukin** [comment]
image
small note
I couldn't find these red stake pools in the list for the next step. May not be an issue at all

**Comment by daniloprates** [comment]
> small note I couldn't find these red stake pools in the list for the next step. May not be an issue at all

@a-rukin most of delegation stuff is hardcoded ATM. I'm integrating the first API endpoint just now.

**Pull request merged**